<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Transferencias</class>
 <widget class="QDialog" name="Transferencias">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>322</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(94, 92, 100);</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>141</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../images/routeImagenes.qrc">:/xy/logoRioCeballos.svg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>30</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tlwg Typewriter</family>
     <pointsize>10</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Transferencias bancarias</string>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Ingrese numero de documento</string>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="tipoDocumento">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <item>
    <property name="text">
     <string>-Seleccione una opcion</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>DNI - Documento nacional de identidad</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>CUIT / CUIL</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Pasaporte</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="valorDocumento">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="motivoTransferencia">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>110</y>
     <width>141</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <item>
    <property name="text">
     <string>-Seleccione una opcion</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Honorarios</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Alquiler</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Otros</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>190</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Monto</string>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="montoTransferencia">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>220</y>
     <width>141</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>80</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Motivo</string>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>190</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Tipo de moneda</string>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="tipoMoneda">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>141</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <item>
    <property name="text">
     <string>-Seleccione una opcion</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Pesos</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Dolares</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="btnEnviarTransferencia">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>270</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Enviar</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../images/routeImagenes.qrc"/>
 </resources>
 <connections/>
</ui>
